app/assets/javascripts/discourse/app/components/category-permission-row.gjs

```diff
@@ -0,0 +1,166 @@
+import Component from "@glimmer/component";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import DButton from "discourse/components/d-button";
+import concatClass from "discourse/helpers/concat-class";
+import PermissionType from "discourse/models/permission-type";
+import getURL from "discourse-common/lib/get-url";
+import I18n from "discourse-i18n";
+
+const EVERYONE = "everyone";
+
+export default class CategoryPermissionRow extends Component {
+  @service currentUser;
+
+  get everyonePermissionType() {
+    return this.args.everyonePermission?.permission_type;
+  }
+
+  get canReply() {
+    return (
+      this.args.type === PermissionType.CREATE_POST ||
+      this.args.type === PermissionType.FULL
+    );
+  }
+
+  get canCreate() {
+    return this.args.type === PermissionType.FULL;
```

Comment:
What to you think about making it a bit more composed and use @equal

https://github.com/discourse/discourse/blob/main/plugins/poll/assets/javascripts/discourse/components/poll-breakdown-option.gjs#L25

Not fully sure about that idea is we can do this to `canCreate` and cannot for `canReply` so maybe for consistency it is better to keep it like that

Reply:
Yes not sure, I do think for consistency it's slightly better to keep them separate